openstack-mixin/targets.libsonnet

```diff
@@ -0,0 +1,317 @@
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_subnets{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}}'),
+    routers:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_routers{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - total'),
+    routersNotActive:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_routers_not_active{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - inactive'),
+    routerDetails:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_router{%(queriesSelector)s}' % vars
+      )
+      + prometheusQuery.withFormat('table')
+      + prometheusQuery.withInstant(true),
+    ports:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_ports{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - total'),
+    portsLBNotActive:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_ports_lb_not_active{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - load balancer inactive'),
+    portsNoIPs:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_ports_no_ips{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - no IPs'),
+    portDetails:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_port{%(queriesSelector)s}' % vars
+      )
+      + prometheusQuery.withFormat('table')
+      + prometheusQuery.withInstant(true),
+    floatingIPs:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_floating_ips{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - total'),
+    floatingIPsAssociatedNotActive:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'openstack_neutron_floating_ips_associated_not_active{%(queriesSelector)s}' % vars
+      )
+      + panel.timeSeries.queryOptions.withInterval('1m')
+      + prometheusQuery.withLegendFormat('{{instance}} - associated inactive'),
```

Comment:
would it make sense to change this to just `inactive` or add on `active` or something similar to the other target that goes alongside this on the panel.
<img width="681" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/4d83a4a1-0708-4fb7-82d3-410740d2ada4">